services/imageService.js

```diff
@@ -24,6 +24,9 @@ const uploadProfilePicture = async ({ file, userId, coordinates }) => {
       },
     });
     const { public_id: publicId, secure_url: url } = uploadResponse;
+    if (dev) {
+      return { publicId, url };
+    }
     await userModel.updateUserPicture({ publicId, url }, userId);
     return { publicId, url };
```

Comment:
so when will the photo updated? 
But since we are uploading image, where is it being uploaded? and where are we maintaining the entry for it?

Reply:
If dev=false, then the photo will be updated onspot,
if dev=true, then photo will not be updated, instead a photo-verification object will be created, and the photo will be updated only when the photo-verification object is `ACCEPTED` [here](https://github.com/Real-Dev-Squad/website-backend/pull/2003/files#diff-9a5062203df8184c1d64d655f2dcd16eebeea484b834b0f461771b463037ceefR442).